functions/flight_response.py

```diff
@@ -1,53 +1,62 @@
     formatted_response = {
-        'legIds': cheapest_flight_itinerary['legIds'],
-        'pricingOptions': cheapest_flight_itinerary['pricingOptions'],
+        'legIds': cheapest_itinerary['legIds'],
+        'pricingOptions': cheapest_itinerary['pricingOptions'],
         'originAirport': {
-            'originPlaceId': cheapest_flight_details['originPlaceId'],
-            'originAirportName': simple_comprehensify_dict('name', list_of_countries, cheapest_flight_details['originPlaceId']),
-            'originAirportIATA': simple_comprehensify_dict('iata', list_of_countries, cheapest_flight_details['originPlaceId'])
+            'originPlaceId': cheapest_legs['originPlaceId'],
+            'originAirportName': f"{simple_comprehensify_dict('name', list_of_countries, cheapest_legs['originPlaceId'])} Airport",
+            'originAirportIATA': simple_comprehensify_dict('iata', list_of_countries, cheapest_legs['originPlaceId'])
         },
         'destinationAirport': {
-            'destinationPlaceId': cheapest_flight_details['destinationPlaceId'],
-            'destinationAirportName': simple_comprehensify_dict('name', list_of_countries, cheapest_flight_details['destinationPlaceId']),
-            'destinationAirportIATA': simple_comprehensify_dict('iata', list_of_countries, cheapest_flight_details['destinationPlaceId'])
+            'destinationPlaceId': cheapest_legs['destinationPlaceId'],
+            'destinationAirportName': f"{simple_comprehensify_dict('name', list_of_countries, cheapest_legs['destinationPlaceId'])} Airport",
+            'destinationAirportIATA': simple_comprehensify_dict('iata', list_of_countries, cheapest_legs['destinationPlaceId'])
         },
         'operatingCarrier': {
-            'id': cheapest_flight_details['operatingCarrierIds'],
-            'name': [value['name'] for key,value in list_of_airlines.items() for i in cheapest_flight_details['operatingCarrierIds'] if key == i][0],
+            'id': cheapest_legs['operatingCarrierIds'],
+            'name': [value['name'] for key, value in list_of_airlines.items() for i in cheapest_legs['operatingCarrierIds'] if key == i][0],
         },
         'marketingCarrier': {
-            'id': cheapest_flight_details['marketingCarrierIds'],
-            'name': [value['name'] for key,value in list_of_airlines.items() for i in cheapest_flight_details['marketingCarrierIds'] if key == i][0],
+            'id': cheapest_legs['marketingCarrierIds'],
+            'name': [value['name'] for key, value in list_of_airlines.items() for i in cheapest_legs['marketingCarrierIds'] if key == i][0],
+        },
+        'arrivalDateTime': {
+            'date': convert_to_date(cheapest_legs['arrivalDateTime']['day'], cheapest_legs['arrivalDateTime']['month'], cheapest_legs['arrivalDateTime']['year']),
+            'time': convert_to_time(cheapest_legs['arrivalDateTime']['hour'], cheapest_legs['arrivalDateTime']['minute'], cheapest_legs['arrivalDateTime']['second'])
```

Comment:
Is there object destructuring in python?
```
second, minute, hour, day, month, year = (lambda second, minute, hour, day, month, year, **rest: (second, minute, hour, day, month, year, rest))(**cheapest_legs['arrivalDateTime'])
```

Reply:
Alternatively, can assign a variable outside to `arrival = cheapest_legs['arrivalDateTime']`, then call `arrival['hour']`, etc